crates/nu-utils/standard_library/std.nu

```diff
@@ -160,3 +160,25 @@ export def-env "path add" [
         $env.PATH
     }
 }
+
+# print a banner for nushell, with information about the project
+#
+# Example:
+# an example can be found in [this asciinema recording](https://asciinema.org/a/566513)
+export def banner [] {
+$"(ansi green)     __  ,(ansi reset)
+(ansi green) .--\(\)°'.' (ansi reset)Welcome to (ansi green)Nushell(ansi reset),
+(ansi green)'|, . ,'   (ansi reset)based on the (ansi green)nu(ansi reset) language,
+(ansi green) !_-\(_\\    (ansi reset)where all data is structured!
+
+Please join our (ansi purple)Discord(ansi reset) community at (ansi purple)https://discord.gg/NtAbbGn(ansi reset)
+Our (ansi green_bold)GitHub(ansi reset) repository is at (ansi green_bold)https://github.com/nushell/nushell(ansi reset)
+Our (ansi green)Documentation(ansi reset) is located at (ansi green)https://nushell.sh(ansi reset)
+(ansi cyan)Tweet(ansi reset) us at (ansi cyan_bold)@nu_shell(ansi reset)
+Learn how to remove this at: (ansi green)https://nushell.sh/book/configuration.html#remove-welcome-message(ansi reset)
+
+It's been this long since (ansi green)Nushell(ansi reset)'s first commit:
+((date now) - ('2019-05-10 09:59:12-0700' | into datetime))
```

Comment:
You can use the datetime value here instead of using `into datetime`

`(date now) - 2019-05-10T09:59:12-07:00)`

Reply:
ooh yes indeed!

let me switch to this simpler solution :+1:

Reply:
should be simpler in 838ca95ec129003843da939318e47d7f19bf1a79, thanks :relieved: